ci/acceptance-test/tests/cmd_reboot_test.go

```diff
@@ -0,0 +1,34 @@
+// +build acceptance
+
+package tests
+
+import (
+	"strings"
+	"testing"
+	"os/exec"
+	"time"
+
+	"github.com/tidwall/gjson"
+)
+
+func TestCmdReboot(t *testing.T) {
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc")
+	instance_id := strings.TrimSpace(stdout.String())
+
+	waitUntil(t, 5 * time.Minute, func()error {
+		cmd := exec.Command("openvdc", "show", instance_id)
+		buf, err := cmd.CombinedOutput()
+		if err !=nil {
+			return err
+		}
+		result := gjson.GetBytes(buf, "instance.lastState.state")
+		if result.String() == "RUNNING" {
+			return nil
+		}
+		time.Sleep(5 * time.Second)
+		return WaitContinue
+	})
+
+	RunCmdAndReportFail(t, "openvdc", "reboot", instance_id)
```

Comment:
Just running the command and checking the exit status isn't enough. A test needs to be done to check if the instance really has rebooted.

A simple way to do this would be to add a line to `rc.local` that creates a file. Then after reboot, check if the file exists.